### ISSUE TITLE
Install: kata-runtime installation fails on Ubuntu 18.04

### ISSUE THREAD

**Issue by ustiugov** [kata-containers/documentation#485]
My system: 
```
Linux iccluster102 4.15.0-50-generic #54-Ubuntu SMP Mon May 6 18:46:08 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

Here is the error

```
ustiugov@iccluster102:~/faas_study$ sudo -E apt-get -y install kata-runtime kata-proxy kata-shim
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 kata-runtime : Depends: kata-containers-image (= 1.7.0) but it is not going to be installed
                Depends: kata-linux-container (= 4.19.28.39) but it is not going to be installed
                Depends: kata-proxy (= 1.7.0) but 1.7.0-25 is to be installed
                Depends: kata-shim (= 1.7.0) but 1.7.0-23 is to be installed
                Depends: kata-ksm-throttler (= 1.7.0) but it is not going to be installed
                Depends: qemu-lite (= 2.11.0+git.87517afd72) but it is not going to be installed
                Depends: qemu-vanilla (= 2.11.2+git.0982a56a55) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

Interestingly, about a week ago it was working fine...

**Comment by mcastelino** [comment]
Can you try using kata-deploy for now which works. I think something broke w.r.t packaging

tl;dr;
```
docker run -v /opt/kata:/opt/kata -v /var/run/dbus:/var/run/dbus -v /run/systemd:/run/systemd -v /etc/docker:/etc/docker -it katadocker/kata-deploy kata-deploy-docker install
```

https://github.com/kata-containers/packaging/blob/master/kata-deploy/README.md#docker-quick-start

**Comment by ustiugov** [comment]
Thanks! It works for me when I run
```
docker run --runtime=kata-qemu -itd alpine
```
However, for now just out of curiosity, I tried running with `kata-fc` runtime and it fails:
```
docker: Error response from daemon: OCI runtime create failed: file /usr/libexec/kata-containers/kata-proxy does not exist: unknown.
```
Is it a reliable way of installing kata containers? Also, does `kata-qemu` there stands for QEMU-lite or regular QEMU?

**Comment by jcvenegas** [comment]
@ustiugov hi, thanks for open the issue, this is related with https://github.com/kata-containers/packaging/pull/532 . I just created a job to use that uses the patch and fix the packages. 
 
In one or two hours this would be fixed.

**Comment by ustiugov** [comment]
I tried this installation option. FYI, it installs only regular QEMU and not QEMU lite. The rest of runtimes (NEMU, FC) do not start (see the error I mentioned above).

**Comment by ustiugov** [comment]
According to my test on Ubuntu 18 last week, the issue is gone. However, how can I check the version of kata containers that are installed with `apt`?

**Comment by grahamwhaley** [comment]
Hi @ustiugov , I'm not really a distro or packaging person, so maybe @jcvenegas @jodh-intel @marcov can give you more accurate answers, but...

If you are asking to check which version was installed, you can as the runtime itself:

```bash
$ kata-runtime kata-env
```

or if you want to ask `apt`, I think either of these might tell you what you need:

```bash
$ apt list kata-runtime
$ apt-cache policy kata-runtime
```

standard `apt` stuff

**Comment by jodh-intel** [comment]
Or `dpkg -l | grep kata` to see what you actually have installed.

Or just run `kata-collect-data.sh` which does all that for you ;)

**Comment by marcov** [comment]
This issue was reported also [here](kata-containers/kata-containers#44), and it was fixed last week.

@ustiugov let us know if things work now for you too.

**Comment by marcov** [comment]
sorry, I should have read all the comments...

> According to my test on Ubuntu 18 last week, the issue is gone. However, how can I check the version of kata containers that are installed with apt?

I would use `apt-cache showpkg kata-runtime`

**Comment by ustiugov** [comment]
thank you, the issue can be closed
